community/modules/scripts/spack-install/variables.tf

```diff
@@ -157,12 +196,19 @@ EOT
 }
 
 variable "caches_to_populate" {
-  description = <<EOT
+  description = <<-EOT
+  DEPRECATED
+
+  The following `commands` can be used to populate a cache:
+
+  ```
+  MIRROR_URL=gs://my-bucket
+  spack buildcache create --mirror-url $MIRROR_URL -af \$(spack find --format /{hash});
```

Comment:
IIUC the command you suggest is not yet in a release version of spack. Correct?

I think I would want to stick with the released version since that is what we install by default. Do you know if there is backwards compatibility?